db/structure.sql

```diff
@@ -244,10 +244,10 @@ CREATE TABLE public.device_story_devices (
 
 
 --
--- Name: device_story_devices_id_seq; Type: SEQUENCE; Schema: public; Owner: -
+-- Name: device_stories_id_seq; Type: SEQUENCE; Schema: public; Owner: -
```

Comment:
I'm surprised that the names of all of the sequences and indexes also changed because of the `rename_table` migration. I will have to verify that behavior in dev.

Reply:
Because we use `structure.sql` approach and not `schema.rb`, I prefer to change it manually, otherwise, it will change entire `structure.sql` with all `autoincrement`, `version`, and other things. Btw, do you know why `structure.sql` vs `schema.rb` ❓

Reply:
I don't really understand what you are saying.

Are you saying that you manually edited `structure.sql`?

Reply:
Yes